ledger/src/blockstore.rs

```diff
@@ -313,6 +313,10 @@ impl Blockstore {
         self.db
     }
 
+    pub fn db_ref(&self) -> &Arc<Database> {
```

Comment:
I'm leaning towards getting rid of this function (or making it private) and replacing with:

1. get_write_batch
2. write_batch

What do you think?

Reply:
Cleaner. I like it. Fixed.